nibabel/parrec.py

```diff
@@ -609,7 +609,7 @@ def __getitem__(self, slicer):
                                  'F')
         # Broadcast scaling to shape of original data
         slopes, inters = self._slice_scaling
-        fake_data = strided_scalar(self._shape)
```

Comment:
Yup, but that creates a writeable array. Unsetting the writeable flag was the needed trick.